packages/core/src/components/Autocomplete/index.tsx

```diff
@@ -608,7 +622,11 @@ export default class Autocomplete<T extends Item> extends React.Component<Props<
   };
 
   renderLoading = () => (
-    <MenuRow>{renderElementOrFunction(this.props.renderLoading) || <Loader inline />}</MenuRow>
+    <MenuRow>
+      <Spacing horizontal={1}>
```

Comment:
Any reason why this is 1 but error above and no results below is 0.5?

Reply:
Honestly it's basically just what looked best, went over this with @kenchendesign as well. Here's loading:
<img width="244" alt="Screen Shot 2019-07-18 at 3 15 24 PM" src="https://user-images.githubusercontent.com/8676510/61495850-0b721280-a96f-11e9-9416-653399864eaf.png">

Vs Failed
<img width="225" alt="Screen Shot 2019-07-18 at 3 15 30 PM" src="https://user-images.githubusercontent.com/8676510/61495852-0d3bd600-a96f-11e9-965e-4181d16a455e.png">

Reply:
👍